tests/index_test.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit test for the index.py file."""
+
+import unittest
+from datetime import datetime, timedelta, timezone
+import json
+from unittest.mock import patch, mock_open, Mock
+import requests_mock
+import requests
+from typing import Dict, Any
+from src import index
+from src import github_services
+
+
+class GenerateMessageTests(unittest.TestCase):
+    """test generate message function."""
+
+    def test_generate_message(self):
+        
+        self.test_template = "{{ username }}\n{{ pr_list }}"
```

Comment:
Use single quotes here and elsewhere, for strings.

Reply:
done